src/AWS.Deploy.CLI/Commands/TypeHints/ExistingSubnetsCommand.cs

```diff
@@ -37,25 +36,48 @@ private async Task<List<Subnet>> GetData(Recommendation recommendation, OptionSe
             return await _awsResourceQueryer.DescribeSubnets(vpcId);
         }
 
-        public async Task<List<TypeHintResource>?> GetResources(Recommendation recommendation, OptionSettingItem optionSetting)
+        public async Task<TypeHintResourceTable> GetResources(Recommendation recommendation, OptionSettingItem optionSetting)
         {
             var subnets = await GetData(recommendation, optionSetting);
-            return subnets.Select(subnet => new TypeHintResource(subnet.SubnetId, subnet.SubnetId)).ToList();
+
+            var resourceTable = new TypeHintResourceTable
+            {
+                Columns = new List<TypeHintResourceColumn>()
+                {
+                    new TypeHintResourceColumn("subnetId", "Subnet Id"),
+                    new TypeHintResourceColumn("vpcId", "VPC Id"),
+                    new TypeHintResourceColumn("az", "Availability Zone")
+                }
+            };
+
+            foreach (var subnet in subnets.OrderBy(subnet => subnet.VpcId))
+            {
+                var row = new TypeHintResource(subnet.SubnetId, subnet.SubnetId);
+                row.ColumnValues["subnetId"] = subnet.SubnetId;
+                row.ColumnValues["vpcId"] = subnet.VpcId;
+                row.ColumnValues["az"] = subnet.AvailabilityZone;
```

Comment:
Based on my previous comment, if we get rid of the Id and just use the field name, can't we deduce the Column names based on the Keys in the dictionary? Making lines 45-50 obsolete?

Reply:
My reply/fix to https://github.com/aws/aws-dotnet-deploy/pull/603#discussion_r898006860 kind of changed this one, but have dropped the Ids/dictionaries.